README.md

```diff
@@ -135,6 +135,12 @@ $ curl -L https://git.io/apk-add-from-url | sh -s -- https://github.com/k1LoW/gh
 $ brew install k1LoW/tap/gh-grep
 ```
 
+**macports:**
+
+```console
+$ sudo port install gh-grep
+```
+
```

Comment:
@herbygillot Thank you !!!

I'd like to add your name to the documentation for the MacPorts package, since I cannot maintain it.

```suggestion
Maintainer: @herbygillot
```